suites/libraries/cms/console/CoreUpdateCommandTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * @package	    Joomla.UnitTest
+ * @subpackage  Version
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license	    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Test class for UpdateCoreCommandTest
+ *
+ * @package     Joomla.UnitTest
+ * @subpackage  Version
+ * @since       4.0
+ */
+
+
+use Joomla\CMS\Application\ConsoleApplication;
+use Joomla\Input\Cli;
+use Joomla\CMS\Console\UpdateCoreCommand;
+
+
+
+class CoreUpdateCommandTest extends \PHPUnit\Framework\TestCase
+{
+	/**
+	 * Object under test
+	 *
+	 * @var    UpdateCoreCommand
+	 * @since  3.0
```

Comment:
@since should be 4.0 here and all the other places

Reply:
okay